include/swift/AST/DiagnosticsParse.def

```diff
@@ -686,6 +686,18 @@ ERROR(sil_witness_assoc_conf_not_found,none,
 ERROR(sil_witness_protocol_conformance_not_found,none,
       "sil protocol conformance not found", ())
 
+// SIL differentiability witnesses
+ERROR(sil_diff_witness_expected_keyword,PointsToFirstBadToken,
+      "expected '%0' in differentiability witness", (StringRef))
+ERROR(sil_diff_witness_expected_parameter_list,PointsToFirstBadToken,
+      "expected an comma-separated list of parameter indices, e.g. (0, 1)", ())
+ERROR(sil_diff_witness_expected_rsquare,PointsToFirstBadToken,
+      "expected ']' to end 'differentiable' attribute", ())
```

Comment:
Bad wording. Looks like it's copied from `[differentiable]` parsing.

Reply:
That diagnostic is actually not used. Removed in https://github.com/apple/swift/pull/27487/commits/771143ae652d5b9cc49fc152863b4d133ee9c9c2.